docs/Doxyfile

```diff
@@ -0,0 +1,2494 @@
+# Doxyfile 1.8.13
```

Comment:
I'd suggest not to blindly generate the docs from all files in the project but rather add the files that actually contain doxygen comments manually one by one to the doxygen invocation. Having blindly generated docs is unneededly verbose and essentially useless. See the HPX documentation toolchain for examples (https://github.com/STEllAR-GROUP/hpx/blob/master/docs/CMakeLists.txt)